### ISSUE TITLE
The contrast of the build number in the footer does not meet the WCAG 2.2 AA criteria

### ISSUE THREAD

**Issue by Rouault108** [Rouault108/rouault#5]
<img width="526" height="78" alt="Image" src="https://github.com/user-attachments/assets/97cd419a-ceb8-43f9-85dd-4b16ef5f83f2" />
